libraries/drivers_framework_rails.rb

```diff
@@ -9,7 +9,7 @@ class Rails < Drivers::Framework::Base
         migrate migration_command deploy_environment assets_precompile assets_precompilation_command
         envs_in_console
       ]
-      packages debian: 'zlib1g-dev', rhel: 'zlib-devel'
+      packages debian: %w[libxml2-dev tzdata zlib1g-dev], rhel: %w[libxml2-devel tzdata zlib-devel]
```

Comment:
@ajgon This is worth pointing out. In my attempts to create an integration test with a very simple Rails app (no DB even!) I needed these two packages `libxml2-dev` and `tzdata` because they are core requirements of Rails 4.x. I think it makes sense to add them here because (a) everyone needs them and (b) they are very standard libraries for the servers that people will be building using `opsworks_ruby` anyway.